packages/firestore/exp/src/api/reference.ts

```diff
@@ -65,7 +63,7 @@ export function getDoc<T>(
   const ref = cast<DocumentReference<T>>(reference, DocumentReference);
   const firestore = cast<Firestore>(ref.firestore, Firestore);
   return getFirestoreClient(firestore).then(async firestoreClient => {
-    const viewSnapshot = await getDocViaSnapshotListener(
+    const viewSnapshot = await firestoreClient.getDocumentFromLocalCache(
```

Comment:
Why does this use `getDocumentFromLocalCache` and not `getDocumentViaSnapshotListener` like the other refactorings in this file?

Reply:
This is a bug. Thanks for catching.